docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,80 @@
+
+The Target Framework Moniker (TFM) for .NET Standard is `netstandard`. At the release of .NET Core 1.0, there will be several `netstandard` versions, not just a 1.0 version. There was a strong desire to create a bridge between Portable Class Libraries (PCL) and .NET Standard.  To establish the compatibility bridge, .NET Standard was pre-versioned to overlap with existing .NET platforms in a compatible way with PCL. .NET Standard versions 1.0 through 1.4 represent the overlap. The .NET Core 1.0 platform implements all .NET Standard versions up to and including 1.5.
+
+PCL and .NET Standard both enable binary sharing but define the applicable APIs in different ways. .NET Standard is the new binary code sharing mechanism going forward.
+
+
+
+
+.NET Core and .NET Standard Versioning
+======================================
+
+The .NET Standard will not adopt the version numbers of .NET Core. They are logically separate. There are three major scenarios to consider.
+
+The following sections discuss API additions. It assumes that the additions are to libraries that are referenced by `NETStandard.Library` metapackage.
+
+Adding APIs to .NET Core
+------------------------
+
+When APIs are added to .NET Core, they will typicall be added .NET Standard. Both .NET Core and .NET Standard will version. The APIs might be added to other .NET implementations at the same time, such as the .NET Framework. 
+
+Adding APIs to another .NET Implementation
+------------------------------------------
+
+When APIs are added to the .NET Framework (for example), but not .NET Core, the .NET Standard will version, but .NET Core will not.
+
+The next time .NET Core ships, it will likely add those new APIs such that it now supports that higher .NET Standard version. It can also add additional APIs at the same time, which could create a new .NET Standard version.
+
+.NET Core ships a Major Version
+-------------------------------
+
+When .NET Core ships a 2.0 version, the .NET Standard will be updated (assuming new APIs were added), but will stay with is 1.x versioning scheme. 
+
+We intend to move to a 2.0 version of .NET Standard only for very significant changes. We have defined a non-exhaustive set of cases where we would move to `netstandard2.0`, none of which we intend to do:
+
+- Drop APIs.
+- Add a major new feature that is potentially breaking for existing runtimes (for example, array slices).
```

Comment:
More detail? (Array slices being breaking implies support for forward compatibility, doesn't it?)